composer.json

```diff
@@ -34,6 +35,7 @@
         "test": "vendor/bin/phpunit"
     },
     "require-dev": {
+        "ext-yaml": "*",
```

Comment:
I'd put `ext-yaml` to [`suggest`](https://getcomposer.org/doc/04-schema.md#suggest), it is not required to use the entire package, only to load YAML files.

Reply:
Yes but it is required to run the tests, so it must be in the `require-dev` section.

Reply:
This means Composer won't alert if the user installs the package with `--no-dev`, so we might have an issue here.